common/djangoapps/student/middleware.py

```diff
@@ -15,22 +15,15 @@ class UserStandingMiddleware(object):
     """
     def process_request(self, request):
         user = request.user
-        try:
-            user_account = UserStanding.objects.get(user=user.id)
-            # because user is a unique field in UserStanding, there will either be
-            # one or zero user_accounts associated with a UserStanding
-        except UserStanding.DoesNotExist:
-            pass
-        else:
-            if user_account.account_status == UserStanding.ACCOUNT_DISABLED:
-                msg = _(
-                    'Your account has been disabled. If you believe '
-                    'this was done in error, please contact us at '
-                    '{support_email}'
-                ).format(
-                    support_email=u'<a href="mailto:{address}?subject={subject_line}">{address}</a>'.format(
-                        address=settings.DEFAULT_FEEDBACK_EMAIL,
-                        subject_line=_('Disabled Account'),
-                    ),
-                )
-                return HttpResponseForbidden(msg)
+        if user.username in UserStandingConfig.current().disabled_list:
```

Comment:
Again, fine for 10s of users - not great for 1000s of users. The whitespace-separated list will break down long before this search becomes too costly. If we really cared now, a `SortedList` could be used.